noir/noir-repo/tooling/noir_js_backend_barretenberg/package.json

```diff
@@ -42,7 +42,7 @@
     "lint": "NODE_NO_WARNINGS=1 eslint . --ext .ts --ignore-path ./.eslintignore  --max-warnings 0"
   },
   "dependencies": {
-    "@aztec/bb.js": "0.35.1",
```

Comment:
I noticed in this sync https://github.com/AztecProtocol/aztec-packages/pull/5935 we didn't switch back to a portal. I have brought the portal back as to not stump someone making breaking changes

Reply:
For some reason @aztec/bb.js is failing to resolve with the portal when it was before and now the `noir-packages-test` workflow is failing. I have switched it back to the pinned version for now as I want to see what results the Brillig stdlib provides for benchmarking.